R/aws.alexa.package.R

```diff
@@ -31,27 +31,70 @@ NULL
 #' @param query query list 
 #' @param key A character string containing an AWS Access Key ID. The default is retrieved from \code{Sys.getenv("AWS_ACCESS_KEY_ID")}.
 #' @param secret A character string containing an AWS Secret Access Key. The default is retrieved from \code{Sys.getenv("AWS_SECRET_ACCESS_KEY")}.
+#' @param verbose A logical indicating whether to be verbose. Default is given by \code{options("verbose")}.
+#' @param session_token Optionally, a character string containing an AWS temporary Session Token. If missing, defaults to value stored in environment variable \env{AWS_SESSION_TOKEN}.
+#' @param region A character string containing the AWS region. If missing, defaults to \dQuote{us-west-1}.
+#' @param headers A list of request headers for the REST call. 
 #' @param \dots Additional arguments passed to \code{\link[httr]{GET}}.
 #' @return list
 
-alexa_GET <- function(query, key = Sys.getenv("AWS_ACCESS_KEY_ID"),
-                            secret = Sys.getenv("AWS_SECRET_ACCESS_KEY"), ...) {
+alexa_GET <- function(query, 
+                      key = Sys.getenv("AWS_ACCESS_KEY_ID"),
+                      secret = Sys.getenv("AWS_SECRET_ACCESS_KEY"), 
+                      verbose = getOption("verbose", FALSE),
+                      session_token = NULL,
+                      region = 'us-west-1',
```

Comment:
Why `us-west-1` and not the default `us-east-1`?

Reply:
Not sure if this is related to my account only, but I get #403 in other regions. If you can confirm it works in `us-east-1` if happy change this :)

Example:

```
> url_info(url = "google.com", region = 'us-east-1')
BŁĄD: HTTP failure: 403
```

Reply:
ok. will merge for now. will investigate this post-merge.